html/register-user/available-user.php

```diff
@@ -0,0 +1,17 @@
+<?php
+require_once('../db/connection.php');
+$uId = $_POST['user_id'];
+$res = $conn->query('select * from user');
+$result = "";
+if ($res->num_rows > 0) {
+    while ($row = $res->fetch_assoc()) {
+        if ($uId == $row["user_id"]) {
+            $result = "user_id already exists.";
+            ?>
```

Comment:
This is closing tag of opening tag which written in 1st line, I think it alignment should be same for better view.